examples/kubernetes/orchestrator-up.sh

```diff
@@ -12,6 +12,10 @@ sed_script=""
 for var in service_type; do
   sed_script+="s,{{$var}},${!var},g;"
 done
+
+# Create configmap from orchestrator docker config file
+$KUBECTL create --namespace=$VITESS_NAME configmap orchestrator-conf --from-file="../../docker/orchestrator/orchestrator.conf.json"
```

Comment:
If you specify the path starting at `$script_root` (defined above), the script will still work if you run it from a different working directory.